packages/ds-medicare-gov/src/styles/components/_SimpleFooter.scss

```diff
@@ -59,16 +59,9 @@
     gap: 0 $spacer-2;
   }
 
-  @media screen and (min-width: $media-width-sm) {
-    flex-direction: column;
```

Comment:
remove because the extra small breakpoint is already set to `column`, so this isn't doing anything